assets/clp/Dockerfile

```
@@ -0,0 +1,18 @@
+
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    htop \
+    jq \
+    python3-venv \
+    rsync \
+    sqlite3 \
+    tmux \
+    vim
+
+COPY --from=0 /clp/clp-s /clp/clp-s
```

Hadolint (2.12.0)

[warning] `COPY --from` should reference a previously defined `FROM` alias

(DL3022)